modules/core/src/core-layers/text-layer/multi-icon-layer/multi-icon-layer.js

```diff
@@ -23,8 +23,8 @@ import IconLayer from '../../icon-layer/icon-layer';
 import vs from './multi-icon-layer-vertex.glsl';
 
 const defaultProps = {
-  getIndexOfIcon: x => x.index || 0,
-  getNumOfIcon: x => x.len || 1,
+  getShiftInQueue: x => x.shift || 0,
```

Comment:
use offset _instead_ of _shift_ for consistency with the _getPixelOffset_ API?
(unless other multi-icon-layer all uses _.shift_)

Reply:
The naming is really confusing. Basically `shift` is applied before scaling to font size, and `pixelOffset` is applied as-is.